linux_os/guide/services/nfs_and_rpc/nfs_configuring_clients/mounting_remote_filesystems/mount_option_krb_sec_remote_filesystems/ansible/shared.yml

```diff
@@ -5,13 +5,21 @@
 # disruption = medium
 
 - name: "Get nfs and nfs4 mount points, that don't have Kerberos security option"
-  shell: grep -E "[[:space:]]nfs[4]?[[:space:]]" /etc/fstab | grep -v "sec=krb5:krb5i:krb5p" | awk '{print $2}'
+  shell: |
+    set -o pipefail
+    grep -E "[[:space:]]nfs[4]?[[:space:]]" /etc/fstab | grep -v "sec=krb5:krb5i:krb5p" | awk '{print $2}'
+  args:
+    warn: False
```

Comment:
Looking through https://github.com/ansible/ansible-lint/issues/497, should ``executable: /bin/bash`` be added to args? e.g.: 
`````
  shell: |
    set -o pipefail
    php -v | grep -F '{{ php_version }}'
  args:
    executable: /bin/bash
`````

Reply:
whenever pipefail is set, yes it should be.